app/assets/stylesheets/argo/workflow_grid.scss

```diff
@@ -0,0 +1,8 @@
+.workflow-grid-table {
+  td a span.zero {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4